hack/unwanted-dependencies.json

```diff
@@ -180,21 +148,51 @@
       "go.opencensus.io": [
         "github.com/Microsoft/hcsshim"
       ],
+      "google.golang.org/appengine": [
+        "github.com/prometheus/client_golang"
+      ],
       "golang.org/x/exp": [
+        "github.com/Microsoft/hcsshim",
         "github.com/antlr4-go/antlr/v4",
+        "github.com/cilium/ebpf",
+        "github.com/containerd/cgroups/v3",
```

Comment:
can we open an issue upstream for github.com/containerd/cgroups/v3 to stop using the unstable exp module?

Reply:
Took a brief look at this. Seems like the indirect reference to this in containerd/cgroups is also coming from google/x/sys and getting rid of it from google/x/sys seems a little convoluted. Should I open a bug there instead?

Reply:
not sure what you mean by google/x/sys ... do you mean `golang.org/x/sys v0.23.0`?

that doesn't seem to list *any* dependencies in https://github.com/golang/sys/blob/v0.23.0/go.mod

Reply:
https://github.com/cilium/ebpf/blob/625b0a910e1ba666e483e75b149880ce3b54dc85/go.mod#L9 coming from here

Reply:
that's super unfortunate that dependency is spreading into other modules ... this was why we asked them to stop using it in https://github.com/cilium/ebpf/issues/1095#issuecomment-1648293305 :-/